apps/Gruntfile.js

```diff
@@ -799,6 +822,12 @@ testsContext.keys().forEach(testsContext);
     'karma:integration'
   ]);
 
+  grunt.registerTask('scratchTest', [
+    'preconcat',
+    'concat',
+    'karma:scratch',
+  ]);
+
```

Comment:
@islemaster suggested this.  The integration tests were failing when running all together because Scratch wanted to load a different version of Blockly into the global window vs. our apps.  Scratch tests would pass, but any later integration tests would fail.